Meadowlark-js/backends/meadowlark-mongodb-backend/test/integration/Config.ts

```diff
@@ -15,7 +15,12 @@ export const setupConfigForIntegration = async () => {
     return;
   }
 
-  const path = join(process.cwd(), '.env');
+  // First look for .env file paired with this Config.ts file
+  let path = join(__dirname, '.env');
```

Comment:
allow for different .envs in each backend rather than force one big .env at root

Reply:
I really like this